Makefile

```diff
@@ -1,23 +1,24 @@
+app_dir='browser'
 
 .PHONY: default
 default: makedb
 
 .PHONY: cleanpositionthumbs
 cleanpositionthumbs:
-	find . -name '*thumbnail*.gif' | xargs rm
+	find $(app_dir) -name '*thumbnail*.gif' | xargs rm
 
 .PHONY: cleanthumbs
 cleanthumbs: cleanpositionthumbs
-	find . -name '*thumbnail*.png' | xargs rm
-	rm -rf protected/images
+	find $(app_dir) -name '*thumbnail*.png' | xargs rm
+	rm -rf $(app_dir)/protected/images
```

Comment:
Oh, the "-rf" is so that if the target is run and the directory is missing then it will not produce an error.